src/material/badge/badge.spec.ts

```diff
@@ -330,3 +381,15 @@ class NestedBadge {}
     <ng-template matBadge="1">Notifications</ng-template>`,
 })
 class BadgeOnTemplate {}
+
+@Component({template: `<mat-icon matBadge="1">home</mat-icon>`})
+class IconWithBadge {}
+
+@Component({template: `<mat-icon matBadge="1" aria-hidden="false">home</mat-icon>`})
+class IconWithBadgeFalseAriaHidden {}
```

Comment:
Could we rename these to have the boolean value come after the property? E.g.
`BadgeOnIconWithAriaHiddenFalse`